assets/js/activitypub-admin.js

```diff
@@ -0,0 +1,14 @@
+jQuery( function( $ ) {
+	// Accordion handling in various areas.
+	$( '.activitypub-settings-accordion' ).on( 'click', '.activitypub-settings-accordion-trigger', function() {
+		var isExpanded = ( 'true' === $( this ).attr( 'aria-expanded' ) );
+
+		if ( isExpanded ) {
+			$( this ).attr( 'aria-expanded', 'false' );
+			$( '#' + $( this ).attr( 'aria-controls' ) ).attr( 'hidden', true );
+		} else {
+			$( this ).attr( 'aria-expanded', 'true' );
+			$( '#' + $( this ).attr( 'aria-controls' ) ).attr( 'hidden', false );
+		}
+	} );
```

Comment:
```suggestion
	} );
	
	$(document).on( 'wp-plugin-install-success', function( event, response ) {
		setTimeout( function() {
			debugger;
			$( '.activate-now' ).removeClass( 'thickbox open-plugin-details-modal' );
		}, 1200 );
	} );	
```
To avoid this, we can remove the class here that opens the thickbox.